# request
std_msgs/String cmd
---
# response
bool success
